src/overview/enhancer.js

```diff
@@ -60,8 +60,12 @@ const locationSync = ReduxQuerySync.enhancer({
 const hydrateStateFromStorage = store => {
     const hydrate = (key, action) =>
         browser.storage.local.get(key).then(data => {
-            if (!data[key]) return
-            store.dispatch(action(data[key]))
+            if (key === constants.SHOW_TOOL_TIP) {
+                store.dispatch(action({ isShowTooltip: data[key] }))
```

Comment:
Change the `showTooltip` action's signature to simply just accept a boolean, rather than object with boolean prop, then you don't need to change this